### PR TITLE
Fix float convert error

### PR DESCRIPTION
**Pull request by Xmax1231**

### CODE REVIEW

**Comment by mlouielu** [comment]
Why will it have null value in the column? Maybe you should also check the value first, not using try catch to prevent the error

File: t.py [hunk 1]
```diff
@@ -52,8 +52,7 @@ def querys(session, user):
 
                 data = json.loads(r.text)
                 for d in data['scores']['scores']:
-                    if(u"通識" in d['title'] and
-                       float(d['final_score']) >= 60.0):
+                    if '通識' in d['title'] and d['final_score'] != '' and float(d['final_score']) >= 60.0:
```
Comment (mlouielu) [inline t.py]: I assume the line is over 80 characters, please wrap this condition

Comment (Xmax1231) [reply t.py]:
```python
if ('通識' in d['title'] and d['final_score'] != '' and
        float(d['final_score']) >= 60.0):
```
Like this?

Comment (mlouielu) [reply t.py]: yep

**Pull request merged**